steps/approved-premises/applications/edit-risk-information-rmp.ts

```diff
@@ -3,7 +3,7 @@ import { type Page, expect } from '@playwright/test'
 export const verifyRMPInfoIsAsPerOASys = async (page: Page) => {
     await page.locator('a', { hasText: 'Risk management plan' }).click()
     await expect(page.getByLabel('Key information about current situation')).toContainText(
-        'is currently in the community having received a Adult Custody (voluntary supervision)'
+        /"(\w+) \w+ is currently in the community having received a Adult Custody \(voluntary supervision\) on the (\d+)\\\/(\d+)\\\/(\d+) for 6 months.*/
```

Comment:
Don't think you need the brackets here, as you're not capturing the information
```suggestion
        /"\w+ \w+ is currently in the community having received a Adult Custody \(voluntary supervision\) on the \d+\\\/\d+\\\/\d+ for 6 months.*/
```